tests/StackExchange.Redis.Tests/BasicOps.cs

```diff
@@ -431,6 +431,22 @@ public void ShouldUseSharedMuxer()
             }
         }
 
+        [Fact]
+        public async Task Copy()
+        {
+            using (var muxer = Create())
+            {
+                var db = muxer.GetDatabase();
+                var src = Me();
+                var dest = Me() + "2";
+                _ = db.StringSetAsync(src, "Heyyyyy");
+                var ke1 = db.KeyCopyAsync(src, dest).ForAwait();
+                var ku1 = db.StringGet(dest);
+                Assert.True(await ke1);
+                Assert.True(ku1.Equals("Heyyyyy"));
+            }
+        }
```

Comment:
Let's please test copying to another database - see `GetDedicatedDB` to grab a database in the test suite that won't conflict (and be sure to delete the key there first to make sure we're testing).

We should also be testing the replace case and that it doesn't replace, etc.

Reply:
Having troubles with copying to another database - I don't know why but the test skips (at least not failing)

Reply:
Fixed in latest - was just missing the DB literal in the command message :)